.speakeasy/workflow.yaml

```diff
@@ -1,9 +1,13 @@
 workflowVersion: 1.0.0
+speakeasyVersion: latest
 sources:
     openapi:
         inputs:
             - location: https://api.unkey.dev/openapi.json
+        registry:
+            location: registry.speakeasyapi.dev/sagar-demo/sagar-demo/openapi
 targets:
-    go:
+    unkey-go:
         target: go
         source: openapi
+        output: ./sdk/go
```

Comment:
make this goclient for now

Reply:
Step2 
Move the gen.yaml and gen.lock into sdk/goclient set packageName [github.com/unkeyed/unkey/sdks/goclient](http://github.com/unkeyed/unkey/sdks/goclient)

All of this can be done with configure targets too